src/script/page/LeftSidebar/panels/Conversations/ConversationHeader/ConversationHeader.tsx

```diff
@@ -83,16 +82,23 @@ export const ConversationHeaderComponent: ForwardRefRenderFunction<HTMLInputElem
   };
 
   const onKeyDown = (event: KeyboardEvent<HTMLInputElement>) => {
-    if (event.key === 'Escape') {
-      setSearchValue('');
-    }
-
-    handleEnterDown(event, () => {
-      onSearchEnterClick(event);
-      setIsConversationFilterFocused(false);
-    });
+    handleEscDown(event, () => setSearchValue(''));
+    handleEnterDown(event, () => onSearchEnterClick(event));
   };
 
+  const onSearchShortcut = useCallback(() => {
```

Comment:
if possible moving the function into the useEffect will no longer need to wrap it in useCallback